app/assets/javascripts/issue_renderer.js

```diff
@@ -1,68 +1,62 @@
 function renderIssue(issue, option) {
-  var labels = issue.labels.map(function(label) {
-    return label.name;
-  });
-  var issueType = getIssueType(issue, labels);
-  var selector = issueSelector(issueType);
+  var labelNames = pluckLabelNames(issue.labels);
+
+  var label = issueLabel(labelNames);
+  var columnId = getColumnId(issue, labelNames);
+  var body = sanitizeIssueBody(issue.body);
 
   if (option == "prepend") {
-    $(selector).prepend(issueTemplate(issue, labels));
+    $(columnId).prepend(issueTemplate(issue, body, label, issue.number, issue.title));
```

Comment:
This list of arguments is getting a little long - might want to consider passing an object instead. (could be a larger refactor than what you're doing here)